pkg/api/repository_parameters.go

```diff
@@ -7,3 +7,28 @@ type RepositoryParameterResponse struct {
 	DistributionVersions []config.DistributionVersion `json:"distribution_versions" ` //Versions available for repository creation
 	DistributionArches   []config.DistributionArch    `json:"distribution_arches"`    //Architectures available for repository creation
 }
+
+//Validation request/response
+type RepositoryValidationRequest struct {
+	Name *string `json:"name"`
+	URL  *string `json:"url"` // URL of the remote yum repository
+}
+
+type RepositoryValidationResponse struct {
+	Name GenericAttributeValidationResponse `json:"name"`
+	URL  UrlValidationResponse              `json:"url"`
```

Comment:
Maybe these fields should get comments/documented. Something like "Validation response for repository name". I was a little confused at first. I feel like it's only intuitive from the naming in hindsight.